File: Module2/src/bai6_ke_thua/bai_tap/lop_triangle/Shape.java
package bai6_ke_thua.bai_tap.lop_triangle;

public class Shape {
    private String color = "green";

    public Shape(){

    }

    public Shape(String color){
        this.color = color;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String toString(){
        return "Shape {color = " + color + "}";
    }
}

File: Module5/bai2/thuc_hanh/typescript-is-shape/src/shape.ts
export class Shape {
    private _x: number;

    private _y: number;

    constructor(x: number, y: number) {
        this._x = x;
        this._y = y;
    }

    get y() {
        return this._y;
    }

    set y(y: number) {
        this._y = y;
    }

    get x() {
        return this._x;
    }

    set x(x: number) {
        this._x = x;
    }

    toString(): string {
        return `(x: ${this._x}, y: ${this._y})`;
    }
}